modules/installation-custom-aws-vpc.adoc

```diff
@@ -59,9 +60,6 @@ The installation program modifies your subnets to add the `kubernetes.io/cluster
 * You must enable the `enableDnsSupport` and `enableDnsHostnames` attributes in your VPC, so that the cluster can use the Route 53 zones that are attached to the VPC to resolve cluster's internal DNS records. See link:https://docs.aws.amazon.com/vpc/latest/userguide/vpc-dns.html#vpc-dns-support[DNS Support in Your VPC] in the AWS documentation.
 +
 If you prefer to use your own Route 53 hosted private zone, you must associate the existing hosted zone with your VPC prior to installing a cluster. You can define your hosted zone using the `platform.aws.hostedZone` field in the `install-config.yaml` file.
-ifndef::public[]
-* If you use a cluster with public access, you must create a public and a private subnet for each availability zone that your cluster uses. Each availability zone can contain no more than one public and one private subnet.
-endif::public[]
 
 ifndef::aws-china[]
 If you are working in a disconnected environment, you are unable to reach the public IP addresses for EC2 and ELB endpoints. To resolve this, you must create a VPC endpoint and attach it to the subnet that the clusters are using. The endpoints should be named as follows:
```

Comment:
I think we'd made a mistake here, the endpoint for C2S should be:
```
elasticloadbalancing.<region>.c2s.ic.gov
ec2.<region>.c2s.ic.gov
s3.<region>.c2s.ic.gov
```
@staebler could you help to confirm this? if yes, we need to submit a bug to fix the previous version docs.

Reply:
+1

Reply:
This section is also a bit misleading, or maybe the term "disconnected" is somewhat overloaded. All installations to C2S are disconnected, in that there is no connection to the public Internet. But the user does not need to create the VPC endpoints, as far as I understand it.

Reply:
@staebler Dug into this. The content about creating the endpoints was added per [BZ 1823457](https://bugzilla.redhat.com/show_bug.cgi?id=1823457)/[PR 22057](https://github.com/openshift/openshift-docs/pull/22057). Is the AWS Top Secret region an exception to the requirement? If so, I can remove the content.

Reply:
I am probably wrong. The endpoints are probably needed in C2S. I was confused since I've always used a pre-existing VPC, that likely already has the endpoints set up.

Reply:
In addition to updating the endpoint names here, I have opened a bug to backport these changes to 4.6-4.9[1] This update has me wondering if the GovCloud endpoint names are correct. They are currently listed as follows:
- `ec2.<region>.amazonaws.com`
- `elasticloadbalancing.<region>.amazonaws.com`
- `s3.<region>.amazonaws.com`

If they also need to be updated, I will add the work to the BZ.
[1] https://bugzilla.redhat.com/show_bug.cgi?id=2050304

Reply:
The GovCloud endpoints are correct.